src/retractions.jl

```diff
@@ -1047,6 +1074,15 @@ retract_softmax!(M::AbstractManifold, q, p, X, t::Number)
 
 function retract_softmax! end
 
+"""
+    retract_sasaki!(M::AbstractManifold, q, p, X, t::Number, m::SasakiRetraction)
+
+Compute the in-place variant of the [`SasakiRetraction`](@ref) `m`.
+"""
+retract_pade!(M::AbstractManifold, q, p, X, t::Number, m::SasakiRetraction)
```

Comment:
```suggestion
retract_sasaki!(M::AbstractManifold, q, p, X, t::Number, m::SasakiRetraction)
```

Reply:
Or do I miss something here? The docs say `sasaki` as well, and it is similar to Padé though (since it also still passes down the retraction type),